### ISSUE TITLE
Fusion Shake Is Reduced

### ISSUE THREAD

**Issue by CDarrow** [CDarrow/DXX-Retro#55]
Fusion shake while charging was much reduced between D1x and Rebirth.  Why?  Can we put it back?

**Comment by vLKp** [comment]
Fusion shake is implemented in [GameProcessFrame](https://github.com/dxx-rebirth/dxx-rebirth/blob/4871b17c8f9e8af4cf74bfaf035283413fbc7558/similar/main/game.cpp#L1348) by adjusting the rotational velocity of `ConsoleObject`.  As best I can tell, the only changes made in Rebirth were related to making the shake independent of the game's framerate.  See dxx-rebirth/dxx-rebirth@63d2c4a2cfc98000978943f85f0034c360be2bd1 for the initial change, and dxx-rebirth/dxx-rebirth@8967ae77fefce9419500fc02ff8f14564ab9058f for some later cleanup to FPS-independent processing.  The original shaking was probably happening every visual frame, so it built up faster than when it happens every game tick.  If you want a more severe shake, your best bet is to change the scale factor on the amount added to `rotvel.x` / `rotvel.z`.  For Retro, this is still in its legacy location of [d1/main/game.c](https://github.com/CDarrow/DXX-Retro/blob/master/d1/main/game.c) / [d2/main/game.c](https://github.com/CDarrow/DXX-Retro/blob/master/d2/main/game.c).

[Historical citations point to the Rebirth repository since Retro was imported without history.]

**Comment by CDarrow** [comment]
Yeah, I'm aware -- but that can't be the whole story.  Demo analysis shows it used to shake a lot more than I would expect given that.  I need to see if I can characterize what was happening. 

I suspect the answer, at this point, will be "don't change it" -- we have too much invested in how the game currently plays for even something original like this to have the value it would have two years ago.  But I'll need to ask.

**Comment by ryusei117** [comment]
I just tested the original D1 and D2 engines, DXX-Rebirth, and DXX Retro. As far as I could see, fusion shaking was similar between all of the engines - too similar for me to observe any differences, anyway. Here are the demos I took, in case they can be analyzed in detail. (Oh yeah, the D2X-Rebirth demo has a blooper at the end ... heh heh.)

- [Fusion shake demos](https://dl.dropboxusercontent.com/u/26944122/descent_issues/fusion_shake/fusion_shake.zip)
